hack/build-single-manifests.sh

```diff
@@ -7,19 +7,19 @@ set -o pipefail
 SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
 
 # k4k8s
-kustomize build ./deploy/manifests/base > deploy/single/all-in-one-dbless.yaml
+kustomize build ./deploy/manifests/base > deploy/single-v1/all-in-one-dbless.yaml
 # k4k8s with DB
 kustomize build ./deploy/manifests/postgres \
-  > deploy/single/all-in-one-postgres.yaml
+  > deploy/single-v1/all-in-one-postgres.yaml
```

Comment:
looks like you have empty single-v1, but some data under single-v2 ?

Reply:
Both of them do have content:

```
$ git status
On branch feat/rg-deploy-tooling
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	cli/ingress-controller/ingress-controller
	railgun/hack/generators/controllers/networking/networking
	rbac.md

nothing added to commit but untracked files present (use "git add" to track)

$ git rev-parse HEAD
843cfc61343ad92db3a535d5fed617c764238857

$ ls deploy/single-v1 
all-in-one-dbless-k4k8s-enterprise.yaml  all-in-one-dbless.yaml  all-in-one-postgres-enterprise.yaml  all-in-one-postgres.yaml

$ ls deploy/single-v2
all-in-one-dbless.yaml	all-in-one-enterprise-dbless.yaml  all-in-one-enterprise-postgres.yaml	all-in-one-postgres.yaml
```